components/portal/messages/controllers.js

```diff
@@ -42,11 +42,13 @@ define(['angular'], function(angular) {
         var getMessages = function() {
           messagesService.getAllMessages()
             .then(function(result) {
-              // Ensure messages exist and check for group filtering
-              if (result.messages && result.messages.length > 0) {
-                allMessages = result.messages;
+              if (result) {
```

Comment:
:question: should this be handled at the `messagesService` layer?
replacing `underfined` with `[]`?

:question: could this be labeled with `TODO` or `FIXME` as a reminder that it is a short term fix?